daffodil-test/src/test/resources/org/apache/daffodil/section13/packed/packed.tdml

```diff
@@ -1019,6 +1020,20 @@
       <tdml:error>Parse Error</tdml:error>
       <tdml:error>out of range for type</tdml:error>
     </tdml:errors>
-  </tdml:parserTestCase>
+    </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="packedIntTooLarge" root="int01" model="packedDecimalRangeChecks"
+    description="Using BCD formatted hex to indicate INT_MAX + 1">
+
+    <tdml:document>
+      <tdml:documentPart type="byte">02147483648C</tdml:documentPart>
+    </tdml:document>
+
+    <tdml:errors>
+      <tdml:error>Parse Error</tdml:error>
+      <tdml:error>Error in packed data</tdml:error>
+      <tdml:error>out of range for type</tdml:error>
+    </tdml:errors>
+    </tdml:parserTestCase>
```

Comment:
Can we add a positive test case for this? To show that you can have a large amount of bytes for packed values as long as the result is within range of the type.

Reply:
Done